calyx-py/calyx/queue_data_gen.py

```diff
@@ -71,12 +78,17 @@ def dump_json(num_cmds, no_err: bool, queue_size: Optional[int] = None):
     commands = {
         "commands": {
             "data": (
-                # The `commands` memory has `num_cmds` items, which are all 0, 1, or 2
+                # The `commands` memory has `num_cmds` items, which are all 0, 1, or 2 (or 3 and 4, for nwc policies)
                 no_err_cmds_list(queue_size, num_cmds)
+                if no_err
```

Comment:
Seems like the no_err helper has not been updated for NWC land. That's cool (for now), but please add to the docstring of that helper so that there isn't any confusion!

Reply:
Handled!